type-c-service/src/driver/tps6699x.rs

```diff
@@ -40,7 +40,7 @@ impl<'a, const N: usize, M: RawMutex, B: I2c> Controller for Tps6699x<'a, N, M,
         let interrupts = self.tps6699x.wait_interrupt(false, |_, _| true).await;
 
         for (i, (interrupt, event)) in zip(interrupts.iter(), self.port_events.iter_mut()).enumerate() {
-            trace!("Interrupt {}: {:#X}", i, interrupt);
+            trace!("Interrupt {}: {:#?}", i, interrupt);
```

Comment:
I believe that printing this as hex was intentional since `interrupt` is a bitflag over a `u32`. Using debug would likely print the integer as decimal, which is less useful for bitflags.

Reply:
The print was broken in `~main` due to the type of `interrupt` not implementing `UpperHex` for all targets, likely due to a feature that would be enabled sometimes and not other times. There probably is a way to ensure that trait exists for the type in all build configurations, but this was the least-invasive change in the meantime.